### ISSUE TITLE
Some minor visual issues

### ISSUE THREAD

**Issue by Rukachan** [jaquadro/StorageDrawers#162]
Grass getting inside the drawer
![Grass getting inside the drawer](http://b.1339.cf/bhffpej.png)

Chests and some other blocks have weird colours
![Chests and some other blocks have weird colours](http://b.1339.cf/srxetrl.png)

Item frames
![Item frames](http://b.1339.cf/stomhbs.png)

StorageDrawers version 1.6.2 for 1.7.10

**Comment by jaquadro** [comment]
I won't be able to do anything about the grass.  The grass renders at a random offset within its block, and will clip into any adjacent blocks.

I'll have to do more research on TileEntity rendering, but it's possible I won't be able to fix the lighting on those blocks.

The frame rendering is odd but likely fixable.

**Comment by jaquadro** [comment]
After a bunch of research, I'm able to fix the rendering on some blocks (e.g. the darkened drawers), but not the vanilla chest.  I'm at a real loss to explain why that block is so special, but it completely defies me.  As a bonus, I was able to improve the label rendering across the board, including grass, leaves,etc. finally rendering with correct color.

The frame rendering has been fixed, and drawers will be properly centered and outward-facing.  Also made some slight tweaks to alignment within inventory slots.

Fixes will appear in the next build.
